.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,15 +14,14 @@
 - [ ] This PR does not introduce any regressions.
 - [ ] I certify that this PR is my own entirely original work, or I have permission from and have attributed the relevant authors.
 - [ ] I have agreed to and signed this project's [Contributor License Agreement](https://cla-assistant.io/mastercomfig/team-comtress-2).
-- [ ] This PR only contains changes to the engine and/or core game framework
+- [ ] This PR only contains changes to the engine and/or core game framework.
 - [ ] This PR targets the `community` branch.
 
 <!-- You do NOT have to answer "yes" to the following, but please mark them if relevant -->
 <!-- To tick a checkbox, place an 'x' in it, like so: [x] -->
 - [ ] This change has been filed as an issue.
 - [ ] No other PRs address this.
 - [ ] This PR is as minimal as possible.
-- [ ] This PR does not introduce any regressions.
```

Comment:
Could you keep this and add it where relevant?

Reply:
It's already the second item on the checklist: https://github.com/mastercomfig/team-comtress-2/blob/master/.github/PULL_REQUEST_TEMPLATE.md#checklist
It's a duplicate.